### ISSUE TITLE
Missing accessibility toolbar in some exports

### ISSUE THREAD

**Issue by ignaciogros** [exelearning/exelearning#1978]
The accessibility toolbar is not present in these exports:

- IMS
- SCORM
- ePub

The files should be correctly referenced in the `imsmanifest.xml` files of both the IMS and SCORM exports.
